_posts/en/newsletters/2019-02-12-newsletter.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'Bitcoin Optech Newsletter #33'
+permalink: /en/newsletters/2019/02/12/
+name: 2019-02-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the newest version of LND, briefly
+describes a tool for generating bitcoin ownership proofs, summarizes the
+disclosure of an RPC vulnerability in Bitcoin Core, and links to an
+Optech study about the usability of Replace-by-Fee.  Also included are
+summaries of notable code changes to popular Bitcoin infrastructure
+projects.
+
+## Action items
+
+- **Upgrade to LND 0.5.2:** this minor-version [release][lnd release]
+  fixes bugs related to stability and improves compatibility with other
+  LN software.
+
+## News
+
+- **Tool released for generating and verifying bitcoin ownership
+  proofs:** Blockstream has [released][reserve audit tool] a tool that
+  helps bitcoin custodians, such as exchanges, prove that they control a
+  certain number of bitcoins without creating an onchain transaction.
+  The tool works by creating an almost-valid transaction that contains
+  all of the same information a valid transaction would
+  contain---proving that the transaction creator had access to all of
+  the information necessary to create a spend (e.g. the private keys).
+  The tool is written in the Rust programming language and uses the
+  increasingly-popular BIP174 Partially-Signed Bitcoin Transaction
```

Comment:
Nit: Could drop hyphen and write "increasingly popular" instead of "increasingly-popular".